CHANGELOG.rst

```diff
@@ -54,6 +54,7 @@ Bug Fixes
 - Prevent a deprecation warning due to truth values of Quantity [#2358]
 - Print a warning when heliographic longitude is set to it's default value of 0 [#2480]
 - parse_time now parses numpy.datetime64 correctly. [#2572]
+- Added mocking to mock online tests for noaa.py. [#2900]
```

Comment:
We don't add use this file anymore. Please add a file using: https://github.com/sunpy/sunpy/blob/master/changelog/README.rst

Reply:
What about the type of entry?

Reply:
Misc I think